roles/ansible-test/tasks/init_collection.yaml

```diff
@@ -14,15 +14,26 @@
   shell: "cat galaxy.yml | {{ ansible_test_venv_path }}/bin/yq -y .name | tail -n +1 | head -1"
   register: _collection_name
 
-- name: Setup location of project
+- name: Setup location of project for integration tests
+  vars:
+    _test_type: "integration"
```

Comment:
I _think_ we could just drop _test_type and reference ansible_test_test_command for tests location